### DISCUSSION TITLE
[Feature] Display who shared a picture in shared album

### DISCUSSION THREAD

**Discussion by xathon** [immich-app/immich#7051] [Feature Request]
### The feature

This is already present on the web, however the app doesn't display in the details of an image in a shared album who created the image. In addition to displaying it in the details, it might be nice to display their profile picture in the bottom right corner of the thumbnail.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6197
